chromedp.go

```diff
@@ -385,6 +388,60 @@ func (c *Context) attachTarget(ctx context.Context, targetID target.ID) error {
 	return nil
 }
 
+// Authentication proxy
+func Authentication(login string, password string) Tasks {
+	return Tasks{
+		fetch.Enable().WithPatterns([]*fetch.RequestPattern{{"*", "", ""}}).WithHandleAuthRequests(true),
+		ActionFunc(func(ctx context.Context) error {
+			if login != "" && password != "" {
```

Comment:
Code could be simplified by decreasing nested constructions.
`if login == "" || password == ""{
return nil
}`